client/my-sites/stats/all-time-views-section/style.scss

```diff
@@ -1 +1,2 @@
 @import "calypso/my-sites/stats/modernized-stats-table-styles.scss";
+@import "calypso/my-sites/stats/stats-views/style.scss";
```

Comment:
Do we want to refactor these styles into a single, modularized component in the future?

Reply:
Yes! I will extract the [key container](https://github.com/Automattic/wp-calypso/issues/71762) with styles as separate files. And patch table styles into `modernized-stats-table-styles.scss`.